content/docs/command-reference/run.md

```diff
@@ -228,13 +228,12 @@ $ dvc run -n my_stage './my_script.sh $MYENVVAR'
   `command`.
 
 - `--no-exec` - create a stage file, but do not execute the `command` defined in
-  it, nor cache dependencies or outputs (like with `--no-commit`, explained
-  below). DVC will also add your outputs to `.gitignore`, same as it would do
-  without `--no-exec`. Use `dvc commit` to force committing existing output file
-  versions to cache.
+  it. DVC will still add the outputs to `.gitignore`. Implies `--no-commit`
+  (explained below). If the outputs already exist in the workspace, use
+  `dvc commit` to force caching them.
```

Comment:
it's not about force caching only

Reply:
True, it also creates/updates the lock file. Fixed (using general language). PTAL

Reply:
Although there might be a bug related to this... See iterative/dvc/issues/5034